mindmaps-graql/src/main/java/io/mindmaps/graql/Graql.java

```diff
@@ -36,6 +36,7 @@
 
 public class Graql {
 
+    private Graql() {}
```

Comment:
If you want to be really pedantic you can throw an `UnsupportedOperationException` here to make sure no one can instantiate via reflection